db/models.py

```diff
@@ -1 +1,34 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_TO_CHOOSE = (
+        ("E", "Elf"),
+        ("D", "Dwarf"),
+        ("H", "Human"),
+        ("O", "Ork")
+    )
+    name = models.CharField(max_length=255, unique=True,
+                            choices=RACE_TO_CHOOSE)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255, unique=False)
```

Comment:
Consider setting the 'unique' attribute to True for the 'email' field if each player should have a unique email address. This would prevent duplicate email entries in the database.